### ISSUE TITLE
Allow sentence markup for e.g. talking books

### ISSUE THREAD

**Issue by martinpub** [nlbdev/nordic-epub3-dtbook-migrator#474]
At MTM, we want to keep the Nordic Guidelines validity in our EPUB 3 talking book production. Therefore, we suggest adding flexibility in the validation rules to allow sentence markup where it's currently disallowed:

- [ ] In heading elements, e.g. `<h1 id="h1_1"><span id="st2-1" class="sentence">Chapter 1</span></h1>` should be allowed (currently disallowed by RelaxNG). Note that a corresponding update must be made to the cross-check between nav.xhtml and content document headings. The cross-check should check integrity between the nav.xhtml toc item text and the corresponding heading text, even if this text node is contained in a parent `<span>`
- [ ] In pagebreak elements, e.g. `<div aria-label="2" role="doc-pagebreak" epub:type="pagebreak" id="page-2" class="page-normal"><span id="st2-3" class="sentence">2</span></div>` should be allowed (currently disallowed in [nordic105])
- [ ] We currently expand aria-label sections to "real" headings for talking book production, e.g. `<section id="level1_1" epub:type="cover" aria-label="Omslag">` --> `<section aria-labelledby="aria_label_1" epub:type="cover" id="level1_1"><h1 id="aria_label_1"><span id="st1-1" class="sentence">Omslag</span></h1>`. There is currently no support for this kind of expansion, but we would like it to be allowed for talking books.
- [ ] In verse contexts
- [ ] etc ...

I think the general thing to update is to allow sentence span wherever element text content is allowed. We can discuss this at tomorrow's validation group meeting.

We are working in parallell with tweaking the sentence detection in Pipeline 2 for MTM, so this is partly connected to a moving target. But I think the overall flexibility is possible to discuss even at this point.

Ping @kalaspuffar @AndersEkl

**Comment by kalaspuffar** [comment]
Hi @martinpub 

As usual, when we make this kind of broad changes to add elements of change elements where ever text occurs, it could lead to unforeseen consequences that might make it hard for reading systems to handle the format.

Best regards
Daniel

**Comment by martinpub** [comment]
Yes, indeed we have to be careful. The first question is, should sentence markup be within or outside the scope of Nordic Guidelines. The conclusion from last validation group meeting was that we should consider this. We will return to this after the summer break.
